components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -148,7 +193,11 @@ define([
 
             console.log("Export data: >>" + exportString + "<<");
             if (exportString != "") {
-                FileSaver.saveTextAs(exportString, fileName);
+                if (exportType != "image") {
+                    FileSaver.saveTextAs(exportString, fileName);
+                } else if (exportType == "image" && imageType == "svg") {
+                    FileSaver.saveTextAs(exportString, fileName, 'base64', 'data:image/png;base64');
```

Comment:
I would instead add a FileSaver.saveAsBinary() method, which would set type ("base64 etc etc")  internally. This would make it clear SaveAsText is for text, SaveAsBinary is for binary, and implementational details will be inside the class

(and both SaveAsText() and SaveAsBinary() would call some othe rmethod "SaveAs" which in turn takes the extra required parameter for type)